tests/e2e/helper.ts

```diff
@@ -0,0 +1,34 @@
+import { Network } from "../../src";
+import { AptosConfig } from "../../src/api/aptos_config";
+import { Faucet } from "../../src/api/faucet";
+import { ClientConfig } from "../../src/types";
+
+export const NODE_URL = process.env.APTOS_NODE_URL!;
+export const FAUCET_URL = process.env.APTOS_FAUCET_URL!;
+
+/**
+ * Returns an instance of a FaucetClient with NODE_URL and FAUCET_URL from the
+ * environment. If the FAUCET_AUTH_TOKEN environment variable is set, it will
+ * pass that along in the header in the format the faucet expects.
+ */
+export function getFaucetClient(): Faucet {
```

Comment:
I don't think we need it? faucet is available when initiate Aptos
```
const aptos = new Aptos();
aptos.fundAccount()
```

Reply:
Ah this was suppose to be on my other PR. But will remove it for now